src/scenarios/blazor/pre.py

```diff
@@ -12,16 +12,17 @@
 
 precommands = PreCommands()
 
-if precommands.has_workload:
-    if precommands.framework == 'net6.0' or precommands.framework == 'net7.0':
-        subprocess.run(["dotnet", "workload", "uninstall", "wasm-tools"])
-    else:
-        subprocess.run(["dotnet", "workload", "uninstall", "microsoft-net-sdk-blazorwebassembly-aot"])
+if precommands.framework == 'net6.0' or precommands.framework == 'net7.0':
+    precommands.uninstall_workload("wasm-tools")
+else:
+    precommands.uninstall_workload("microsoft-net-sdk-blazorwebassembly-aot")
 
 setup_loggers(True)
 precommands.new(template='blazorwasm',
                 output_dir=const.APPDIR,
                 bin_dir=const.BINDIR,
                 exename=EXENAME,
                 working_directory=sys.path[0])
-precommands.execute()
+# ensure that we don't use the workload, which would trigger native
+# relinking
+precommands.execute(['-p:WasmNativeWorkload=false'])
```

Comment:
We can already pass this flag by using the `--msbuild` argument and updating the `blazor_scenarios.proj`. Was there a reason that you added this new functionality?

Reply:
I wanted to set here for the project, instead of passing it from whoever is building the project, to ensure that the relinking/not-relinking thing is enforced by the project itself.

Reply:
OK. I think that this is a fine way to do it for now, but I would like to think deeper on how we can do this in a more general way.